src/release/breaking-changes/3-10-deprecations.md

```diff
@@ -336,6 +341,161 @@ Relevant PRs:
+
+**Migration guide**
+
+[In-depth migration guide available]
+
+Code before migration:
+
+```dart
+final AnimationSheetBuilder animationSheet = AnimationSheetBuilder(
+    frameSize: const Size(40, 40)
+);
+
+await tester.pumpFrames(animationSheet.record(
+  const Directionality(
+    textDirection: TextDirection.ltr,
+    child: Padding(
+      padding: EdgeInsets.all(4),
+      child: CircularProgressIndicator(),
+    ),
+  ),
+), const Duration(seconds: 2));
+
+tester.binding.setSurfaceSize(animationSheet.sheetSize());
+
+final Widget display = await animationSheet.display();
+await tester.pumpWidget(display);
+
+await expectLater(
+  find.byWidget(display),
+  matchesGoldenFile('material.circular_progress_indicator.indeterminate.png'),
+);
+```
+
+Code after migration:
+
+```dart
+final AnimationSheetBuilder animationSheet = AnimationSheetBuilder(
+    frameSize: const Size(40, 40)
+);
+
+await tester.pumpFrames(animationSheet.record(
+  const Directionality(
+    textDirection: TextDirection.ltr,
+    child: Padding(
+      padding: EdgeInsets.all(4),
+      child: CircularProgressIndicator(),
+    ),
+  ),
+), const Duration(seconds: 2));
+
+await expectLater(
+  animationSheet.collate(20),
+  matchesGoldenFile('material.circular_progress_indicator.indeterminate.png'),
+);
+```
+
+[In-depth migration guide available]: {{site.url}}/release/breaking-changes/animation-sheet-builder-display
+
+**References**
+
+API documentation:
+
+* [`AnimationSheetBuilder`][]
+
+Relevant PRs:
+
+* Deprecated in [#83337][]
+* Removed in [#129657][]
+
+[`AnimationSheetBuilder`]: {{site.api}}/flutter/flutter_test/AnimationSheetBuilder-class.html
+
+[#83337]: {{site.repo.flutter}}/pull/83337
+[#129657]: {{site.repo.flutter}}/pull/129657
+
+---
+
+---
+
+### flutter_test timeout logic
+
+Package: flutter_test
+Supported by Flutter Fix: no
+
+The following APIs related to timeout logic in tests were deprecated
+in v2.6. There are no replacements, and references should be removed, except for
+the `initialTimeout` parameter of `testWidgets`, which is replaced by using
+`timeout`.
+
+* `TestWidgetsFlutterBinding.addTime`
+* `TestWidgetsFlutterBinding.runAsync` method - `additionalTime` parameter
+* `TestWidgetsFlutterBinding.runTest` method - `timeout` parameter
+* `AutomatedTestWidgetsFlutterBinding.runTest` method - `timeout` parameter
+* `LiveTestWidgetsFlutterBinding.runTest` method - `timeout` parameter
+* `testWidgets` method - `initialTime` parameter
+
+These were found to cause flakiness in testing, and were not in use by tested
+customers.
+
+Since being deprecated, use of these parameters have had no effect on tests, so
+removing references should have no effect on existing code bases.
+
+**Migration guide**
+
+[In-depth migration guide available]
+
+Code before migration:
+
+```dart
+testWidgets('Test', (_) {}, initialTimeout:  Duration(seconds: 5));
+```
+
+Code after migration:
+
+```dart
+testWidgets('Test', (_) {}, timeout:  Duration(seconds: 5));
```

Comment:
```suggestion
testWidgets('Test', (_) {}, timeout:  Timeout(Duration(seconds: 5)));
```

The new `timeout` parameter is actually a `Timeout`, not a direct `Duration` replacement.